.github/actions/setup/action.yml

```diff
@@ -3,26 +3,36 @@ name: Setup environment
 inputs:
   cache:
     description: Enable caching
-    default: "true"
+    default: 'true'
   node:
     description: The Node.js version to install
     required: true
   solana:
     description: The Solana version to install
 
 runs:
-  using: "composite"
+  using: 'composite'
   steps:
     - name: Setup pnpm
       uses: pnpm/action-setup@v3
     - name: Setup Node.js
       uses: actions/setup-node@v4
       with:
         node-version: ${{ inputs.node }}
-        cache: "pnpm"
+        cache: 'pnpm'
     - name: Install dependencies
       run: pnpm install --frozen-lockfile
       shell: bash
+    - name: Install Rust
+      uses: metaplex-foundation/actions/install-rust@v1
+      with:
+        toolchain: stable
+        components: clippy,rustfmt
```

Comment:
We don't use the stable versions of clippy and rustfmt in SPL, may as well avoid installing these if they aren't needed